client/my-sites/media-library/content.jsx

```diff
@@ -136,11 +144,19 @@ module.exports = React.createClass( {
 					);
 					break;
 				default:
-					message = this.translate(
```

Comment:
>I think we should be implementing media exceeding allocated storage

On second thought, I might disagree with myself here. Is the idea that we could eventually migrate existing validation checks to Redux selectors as you've done with overage checking here?